site/docs/aws.md

```diff
@@ -373,8 +373,7 @@ s3://my-table-data-bucket/2d3905f8/my_ns.db/my_table/category=orders/00000-0-5af
 ```
 
 Note, the path resolution logic for `ObjectStoreLocationProvider` is as follows:
-- if `write.object-storage.path` is set, use it
-- if not found, fallback to `write.folder-storage.path`
+- if `write.data.path` is set, use it
 - if not found, use `<tableLocation>/data`
```

Comment:
Given we have a simple path resolution strategy, I think we can just put these listing in a single sentence. Also add 2 warning blocks describing the legacy behaviors:
- before 0.12.0, `write.object-storage.path` must be set
- at 0.12.0, `write.object-storage.path` then `write.folder-storage.path` then `<tableLocation>/data`

Reply:
Thanks Jack! Made the change for the doc.